sdk/core/azure-core/src/test/java/com/azure/core/util/polling/PollingUtilsTests.java

```diff
@@ -64,4 +69,57 @@ public void missingOrDuplicateSlashInPath() {
         // Case 4: both of endpoint and relative path have slash
         assertEquals(expectedPath, getAbsolutePath(relativePathWithSlash, endpointWithSlash, null));
     }
+
+    @Test
+    public void testConvertResponse() {
+        String status = "Succeeded";
+
+        Resource convertOrigin = new Resource().setName("name").setStatus(status);
+
+        PollResult convertResult = PollingUtils.convertResponse(convertOrigin,
+            new DefaultJsonSerializer(),
+            TypeReference.createInstance(PollResult.class)).block();
+
+        assertEquals(status, convertResult.getStatus());
+    }
+
+    private static class PollResult {
+        @JsonProperty(value = "status", access = JsonProperty.Access.WRITE_ONLY)
+        private String status;
+
+        public String getStatus() {
+            return status;
+        }
+
+        public PollResult setStatus(String status) {
+            this.status = status;
+            return this;
+        }
+    }
+
+    private static class Resource {
+        @JsonProperty(value = "name", required = true)
+        private String name;
+
+        @JsonProperty(value = "status", access = JsonProperty.Access.WRITE_ONLY)
```

Comment:
Test will pass if remove "access = JsonProperty.Access.WRITE_ONLY".

Reason is when the property is read-only, Jackson won't serialize it to JSON.